docs/src/snap/howto/cis-hardening/controlplane.md

```diff
@@ -0,0 +1,52 @@
+<!-- markdownlint-disable -->
+## Control Plane Configuration
+### Authentication and Authorization
```

Comment:
what options do we have for inserting text between these? run on headings  look terrible and are difficult to parse.

Reply:
I added a new line between them for the moment. Let's plan work to add text between them next.